src/store/Workspaces/index.ts

```diff
@@ -371,7 +371,6 @@ export const actionCreators: ActionCreators = {
         devWorkspaceClient.changeWorkspaceStatus(workspace.namespace as string, workspace.devfile.metadata.name as string, false);
       } else {
         cheWorkspaceClient.restApiClient.stop(workspace.id);
-        unSubscribeToEnvironmentOutput(workspace.id);
```

Comment:
Is there a reason why this part's removed?

Reply:
I have thoughts that in the case when we start a workspace, then go to workspaces list and stop the starting workspace, then return back to the IDE loader tab - we should see all logs untill the target workspace will be stopped.  Then it will going to unsubscribe here:
https://github.com/eclipse/che-dashboard/pull/189/files/4a5b537822993117c3f899aa77f1b2d453087271#diff-a4a55f8b114e0ad53b1be29dbb5f92d3f9fa4868776569160ea6452e0120c3f9R184

Reply:
Makes sense to me